examples/getting-started/MAC_with_podman/README.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started on a MAC M-Series (arm64-based)
+
+- Install 'brew'
+    ```
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+    ```
```

Comment:
I'd be concerned about encouraging this shell execution from the internet in our documentation, I think it's better to link to the Homebrew installation documentation and then let users decide to do that if they want.